zulipterminal/ui_tools/boxes.py

```diff
@@ -652,7 +652,7 @@ def mouse_event(self, size: urwid_Size, event: str, button: int,
                     col: int, row: int, focus: bool) -> bool:
         if event == 'mouse press':
             if button == 1:
-                self.keypress(size, 'enter')
+                self.keypress(size, next(iter(keys_for_command('ENTER'))))
```

Comment:
You don't have a connected test change for this?

Reply:
My bad, we didn't have any test for this to change. I have added a test now.